pkg/cluster/driver/capi/capi.go

```diff
@@ -309,6 +309,20 @@ func CreateDriver(config *types.Config, clusterConfig *types.ClusterConfig) (dri
 		cdi = string(cdiBytes)
 	}
 
+	// Validate the provider configuration.  For OCI-CCM several pieces of
+	// configuration are required.  Specifically, a compartment, a vcn and
+	// two subnets (which can be the same).  These values are fed into the
+	// OCI-CCM configuration.
+	if clusterConfig.Providers.Oci.Compartment == "" {
+		return nil, fmt.Errorf("the oci provider requires a compartment in the provider with configuration")
```

Comment:
Should OCI be capitalized in these messages?

Reply:
I could go either way on this.  They are lower case because it is the name of the provider rather than the service.  If you prefer them capitalized I'm happy with that.

Reply:
Ok, then leaving as is for now seems fine